daemon/logger/fluentd/fluentd.go

```diff
@@ -285,12 +285,12 @@ func parseAddress(address string) (*location, error) {
 	}
 
 	switch addr.Scheme {
-	case "unix", "unixgram":
+	case "unix":
```

Comment:
Did "unixgram" never work since the underlying driver didn't support it?

Reply:
yes  😅 (at least, see my comment above)